media-plugins/vdr-dvbapi/vdr-dvbapi-2.2.4.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit vdr-plugin-2
+
+DESCRIPTION="VDR Plugin: allows connect VDR to OScam"
+HOMEPAGE="https://github.com/manio/vdr-plugin-dvbapi"
+SRC_URI="https://github.com/manio/vdr-plugin-dvbapi/archive/v${PV}.tar.gz -> vdr-dvbapi-${PV}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="GPL-2"
```

Comment:
It looks like GPL-2+, actually.

Reply:
Hmm... can't follow that. If I run a "diff COPYING /usr/portage/licenses/GPL-2" I find no fundamental differences - so how do you judge that the content of COPYING is GPL2+?

Reply:
although the text itself contains several phrases like "that version or of any later version.." - so maybe /usr/portage/licenses/GPL-2 and COPYING are both GPL-2+? (just a logical thought)

Reply:
Look at the headings in source files.